internal/js_printer/js_printer.go

```diff
@@ -2855,9 +2888,19 @@ func (p *printer) printStmt(stmt js_ast.Stmt) {
 			p.printSpace()
 		}
 		p.printSymbol(s.Fn.Name.Ref)
-		p.printFn(s.Fn)
+		if s.HotModule != nil {
+			name := p.renamer.NameForSymbol(s.HotModule.RefreshFn)
+			refreshSigVarName := "__spinx_refresh_sig_" + name + "__"
```

Comment:
Really should have a helper function for these variable names. prelude / epilog do similar things.